microservices/user/app/handlers/user_feedback.go

```diff
@@ -0,0 +1,123 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/alexandr-io/backend/user/data"
+	"github.com/alexandr-io/backend/user/database/feedback"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type field struct {
+	Name   string `json:"name"`
+	Value  string `json:"value"`
+	Inline bool   `json:"inline"`
+}
+
+type embed struct {
+	Title       string    `json:"title"`
+	Description string    `json:"description"`
+	Timestamp   time.Time `json:"timestamp"`
+	Color       int32     `json:"color"`
+	Fields      []field   `json:"fields"`
+}
+
+type webhook struct {
```

Comment:
The types definition should be placed in the data folder.
This is not mandatory here since theses data are only used within this file